pkg/initialize/run.go

```diff
@@ -60,20 +63,202 @@ func Run(isNonInitCmd bool) error {
 
 // DownloadPolicies clones the policies to a local folder
 func DownloadPolicies() error {
-
+	accessToken := config.GetPolicyAccessToken()
 	policyBasePath := config.GetPolicyBasePath()
-	repoURL := config.GetPolicyRepoURL()
-	branch := config.GetPolicyBranch()
 
 	zap.S().Debug("downloading policies")
-
 	zap.S().Debugf("base directory path : %s", policyBasePath)
-	zap.S().Debugf("policy directory path : %s", config.GetPolicyRepoPath())
-	zap.S().Debugf("policy repo url : %s", repoURL)
-	zap.S().Debugf("policy repo git branch : %s", branch)
 
-	os.RemoveAll(policyBasePath)
+	err := os.RemoveAll(policyBasePath)
+	if err != nil {
+		return fmt.Errorf("unable to delete base folder. error: '%v'", err)
+	}
+
+	if accessToken == "" {
+		return downloadTerrascanPolicies(policyBasePath)
+	}
+
+	return downloadCommercialPolicies(policyBasePath, accessToken)
+}
+
+func downloadCommercialPolicies(policyBasePath, accessToken string) error {
+	err := ensureDir(policyBasePath)
+	if err != nil {
+		return err
+	}
+
+	policyRepoPath := config.GetPolicyRepoPath()
+	err = os.MkdirAll(policyRepoPath, filePermissionBits)
+	if err != nil {
+		return err
+	}
+
+	const apiPath = "/v1/api/app/rules?default=true"
+	environment := config.GetPolicyEnvironment()
+
+	zap.S().Debugf("policy environment : %s", environment)
+	zap.S().Debugf("downloading commercial policies in %s", policyBasePath)
+
+	var client http.Client
+
+	req, err := http.NewRequest(http.MethodGet, environment+apiPath, nil)
+	if err != nil {
+		return fmt.Errorf("error constructing request object. error: '%v'", err)
+	}
+	req.Header.Add("Authorization", "Bearer "+accessToken)
+
+	res, err := client.Do(req)
+	if err != nil || res.StatusCode != 200 {
+		return fmt.Errorf("error downloading commercial policies. error: '%v', response status code: '%d'", err, res.StatusCode)
+	}
+	if res == nil {
+		return fmt.Errorf("error could not download policies, please check your network connection")
+	}
+
+	policies, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return fmt.Errorf("error reading api call response for commercial policies. error: '%v'", err)
```

Comment:
can we change this to `fmt.Errorf("error reading response body, error: %w, err")` ?
please where `%w` to wrap errors.